lightstep/resource_metric_dashboard_test.go

```diff
@@ -15,9 +15,11 @@ import (
 func TestAccMetricDashboard(t *testing.T) {
 	var dashboard client.UnifiedDashboard
 
+	uqlQuery := `metric pagerduty.task.success | filter (kube_app == "pagerduty") | rate | group_by ["cluster-name"], max`
+
 	// missing required field 'type'
-	badDashboard := `
-resource "lightstep_metric_dashboard" "test" {
+	badDashboard := fmt.Sprintf(`
+resource "lightstep_dashboard" "test" {
```

Comment:
same q, this resource has its own tests https://github.com/lightstep/terraform-provider-lightstep/blob/main/lightstep/resource_dashboard_test.go